### PR TITLE
Add a Machine Readable State for container

### PR DESCRIPTION
**Pull request by vdemeester**
See docker/docker#18966

/cc @calavera @MHBauer 

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by mariusGundersen** [comment]
`s/human/machine` :laughing:

**Comment by vdemeester** [comment]
@mariusGundersen damn you're right.. did write it too quick :stuck_out_tongue_closed_eyes:

File: types/types.go [hunk 1]
```diff
@@ -142,6 +142,7 @@ type Container struct {
 	SizeRw     int64 `json:",omitempty"`
 	SizeRootFs int64 `json:",omitempty"`
 	Labels     map[string]string
+	State      string
 	Status     string
```
Comment (dongluochen) [inline types/types.go]: In docker/docker#18966 code or doc, the order is `State` after `Status`. 
```
 	Status     string
 +	State      string
```

Comment (vdemeester) [inline types/types.go]: true :stuck_out_tongue_closed_eyes: But I thought it made more sense to make it alphabetically (but Labels is not so... meh.. :smiley:)

**Comment by dongluochen** [comment]
LGTM.

**Comment by thaJeztah** [comment]
ping @calavera @MHBauer for review / merge :)

**Comment by calavera** [comment]
LGTM

**Pull request merged**